core/src/commonMain/kotlin/com/sourcepoint/mobile_core/Coordinator.kt

```diff
@@ -84,23 +83,27 @@ class Coordinator(
             usnat = campaigns.usnat?.let { SPUserData.SPConsent(consents = state.usNat.consents) }
         )
 
+    @Suppress("Unused")
     constructor(
         accountId: Int,
         propertyId: Int,
         propertyName: SPPropertyName,
         campaigns: SPCampaigns,
-        repository: Repository = Repository(),
-        initialState: State? = null
```

Comment:
@andresilveirah why did you remove `initialState` ? We use it to transfer the state from iOS SDK.

Reply:
@Nevazhnovu we can still pass initial state in the primary constructor. I removed it from the secondary one only.